### ISSUE TITLE
Spade cannot recognize code elements under namespace

### ISSUE THREAD

**Issue by blinkjiang** [codecadwallader/codemaid#572]
### Environment
- Visual Studio version: 2015 enterprise
- CodeMaid version: 10.5
- Code language: C++

### Description
![111](https://user-images.githubusercontent.com/41735672/43257680-0518e006-9103-11e8-8dd2-489bd0949b9e.png)

CodeModelBuilder.cs  line 128
codeElement.Children is **Empty** when codeElement.Kind is vsCMElementNamespace

**Comment by codecadwallader** [comment]
Thanks for reporting the issue and looking into it.  If the API is not able to provide the elements, it makes sense that they would not be available.

**Comment by blinkjiang** [comment]
It seems that the problem only occurs in the cpp file. The corresponding header file is normal, but the CodeElement(StartLine EndLine and other information) from header file seems come from the cpp file. This is weird.

**Comment by codecadwallader** [comment]
Yeah, unfortunately the older EnvDTE APIs have never been super consistent.. more so for C++ than other languages like C# and VB.  Also unfortunately, I haven't heard any plans about C++ moving on to the newer Roslyn APIs/pipeline.

**Comment by richardeakin** [comment]
Confirming that this is still a bug in Visual Studio 2022, Codemain version 12.0. Removing the namespace from my class makes the methods show up in Spade that are within the .cpp class. This really limits the usefulness as almost all of the C++ code that I work with is within namespaces.
